packages/store-ui/src/atoms/Input/Input.stories.mdx

```diff
@@ -0,0 +1,79 @@
+import { Meta, Story, Preview, ArgsTable } from '@storybook/addon-docs/blocks'
+import Input from '.'
+
+<Meta title="Atoms/Input" />
+
+export const colorByState = {
+  default: 'black',
+  success: 'green',
+  error: 'red',
+}
+
+# Default
+
+<Preview>
+  <Story
+    name="Default"
+    args={{
+      placeholder: 'First name',
+    }}
+  >
+    {(args) => (
+      <Input
+        style={{
+          borderColor: colorByState[args.state],
+          borderStyle: 'solid',
+        }}
+        {...args}
+      />
+    )}
+  </Story>
+</Preview>
+
+# <h1 style={{ color: 'green' }}>Success</h1>
+
+<Preview>
+  <Story
+    name="Success"
+    args={{
+      placeholder: 'First name',
+      state: 'success',
+    }}
+  >
+    {(args) => (
+      <Input
+        style={{
+          borderColor: colorByState[args.state],
```

Comment:
can't we add this explicit for having a better preview?
![image](https://user-images.githubusercontent.com/1753396/122988605-4f9a9880-d378-11eb-8d67-7dcc9d8fa533.png)